core/DataAccess/LogAggregator.php

```diff
@@ -274,7 +274,7 @@ public function generateQuery($select, $from, $where, $groupBy, $orderBy, $limit
                         if (stripos($where, 'and ') === 0) {
                             $where = substr($where, strlen('and '));
                         }
-                        $bind = array_slice($bind, 3);
+                        $bind = array();
```

Comment:
We can't just remove 3 entries eg when for roll up reporting there are multiple sites specified. Then `bind` might be actually several idSites plus the start date and end date